containers/kubernetes/reference-content/lb-ingress-controller.mdx

```diff
@@ -1,237 +1,129 @@
 
 <Macro id="iam-requirements" />
 
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
-  - You have [created a Kapsule cluster](/containers/kubernetes/how-to/create-cluster/) and deployed an ingress controller using the Application Library in the [Easy Deploy feature](/containers/kubernetes/how-to/enable-easy-deploy/).
-  - You have downloaded the corresponding [kubeconfig](/containers/kubernetes/how-to/edit-cluster/) file and [kubectl](/containers/kubernetes/how-to/connect-cluster-kubectl/) is working
+  - You have  [set up a Kapsule cluster](/containers/kubernetes/how-to/create-cluster/) , and you have deployed a TRAEFIK2 ingress controller via the application library using the [Easy Deploy function](/containers/kubernetes/how-to/enable-easy-deploy/).
+   - You've obtained the [kubeconfig](/containers/kubernetes/how-to/edit-cluster/) file for the cluster, and [kubectl](/containers/kubernetes/how-to/connect-cluster-kubectl/) is installed on your local machine.
 </Message>
 
-## Exposing the ingress controller through a LoadBalancer service
 
-<Message type="important">
-  You need a functioning Kubernetes Kapsule cluster with an ingress controller deployed with the [Easy Deploy (Application Library) feature](/containers/kubernetes/how-to/enable-easy-deploy/) to follow this tutorial. To deploy your ingress controller, go to the Easy Deploy tab of your existing cluster, create a new deployment, and select the Ingress controller of your choice in the application library.
-</Message>
+## Expose the Ingress Controller using Scaleway Load Balancer
 
-By default, on Kapsule, these ingress controllers are deployed using a [hostPort](https://kubernetes.io/docs/concepts/services-networking/service/). This means that the ingress controller will be accessible on all the machines of your cluster on port 80 and 443.  
-We have decided to do that because the addition of a Load Balancer adds an extra cost for the final user. By using a host port, you will be able to use the ingress objects just after deploying your cluster. We believe that it is a good solution for test and development purposes.  
-We will see in the second part of the tutorial on how to use a Load Balancer to make this production-ready.
+By default, ingress controllers on Kapsule are deployed using a [hostPort](https://kubernetes.io/docs/concepts/services-networking/service/). This means they are accessible on all cluster nodes on ports 80 and 443. This choice was made to prevent additional costs for users during their test&deployment phase. However, for production-readiness, you might want to use a Load Balancer to expose your services on the internet.
```

Comment:
```suggestion
By default, ingress controllers on Kapsule are deployed using a [hostPort](https://kubernetes.io/docs/concepts/services-networking/service/). This means they are accessible on all cluster nodes on ports 80 and 443. This choice was made to prevent additional costs for users during their test & deployment phase. However, for production-readiness, you might want to use a Load Balancer to expose your services on the internet.
```